op-node/p2p/gossip.go

```diff
@@ -250,6 +255,128 @@ func (sb *seenBlocks) markSeen(h common.Hash) {
 	sb.blockHashes = append(sb.blockHashes, h)
 }
 
+// CHANGE(taiko): add preconfBlocks topic validator
+func BuildPreconfBlocksValidator(log log.Logger, cfg *rollup.Config, runCfg GossipRuntimeConfig, blockVersion eth.BlockVersion) pubsub.ValidatorEx {
+	// Seen block hashes per block height
+	// uint64 -> *seenBlocks
+	preconfblockLRU, err := lru.New[uint64, *seenBlocks](1000)
+	if err != nil {
+		panic(fmt.Errorf("failed to set up block height LRU cache: %w", err))
+	}
+
+	return func(ctx context.Context, id peer.ID, message *pubsub.Message) pubsub.ValidationResult {
+		// [REJECT] if the compression is not valid
+		outLen, err := snappy.DecodedLen(message.Data)
+		if err != nil {
+			log.Warn("invalid snappy compression length data", "err", err, "peer", id)
```

Comment:
```suggestion
			log.Warn("invalid snappy compression length data", "peer", id, "err", err)
```

Reply:
This is copied from the original optimism P2P log, i think its ok to keep it.